test/src/main/java/org/locationtech/geowave/test/KuduStoreTestEnvironment.java

```diff
@@ -69,4 +69,10 @@ protected void initOptions(StoreFactoryOptions options) {
     kuduOptions.setKuduMaster("127.0.0.1:7051");
   }
 
+  @Override
+  public int getMaxCellSize() {
```

Comment:
Should we also add this to the other data stores with noted limitations?

Reply:
As of right now, the other datastore work with the original values. I think that can be an incremental change for the other datastores.